pkgs/tools/filesystems/garage/default.nix

```diff
@@ -1,25 +1,45 @@
-{ lib, stdenv, rustPlatform, fetchFromGitea, protobuf, testers, Security, garage }:
+{ lib, stdenv, rustPlatform, fetchFromGitea, openssl, pkg-config, protobuf,
+  testers, Security, garage }:
```

Comment:
```suggestion
{ lib, stdenv, rustPlatform, fetchFromGitea, openssl, pkg-config, protobuf
, testers, Security, garage }:
```

Reply:
Addressed in 654192bd631548f5689102799f22090ab810682e